app/models/bag.rb

```diff
@@ -34,7 +40,7 @@ def bag_name
   # at least one file attached, no duplicate filenames, and a handle pointing to its DSpace record.
   # TODO: add a check for metadata csv
```

Comment:
I think we can remove this todo, since we're building the csv in the `data` method.